lib/board.ex

```diff
@@ -1,57 +1,47 @@
 defmodule Board do
 
-  @markers ["X","O"]
-
-  def create(board_size) do
-    {:x, [["1","2","3"],["4","5","6"],["7","8","9"]], :first_turn}
+  def create do
+    [["1","2","3"],["4","5","6"],["7","8","9"]]
   end
 
-  def place_marker(selected_tile, game_status) do
-    marker = elem(game_status, 0)
-    elem(game_status, 1)
-    |> check_tile_not_taken(selected_tile, marker)
-  end
+  def place_marker(selected_tile, board, player_symbol) do
```

Comment:
_marker_ vs _selected_tile_

I'd personally vote for 'marker', as 'tile' makes me think of bathrooms 😄